[docs] Update saml-google.md (#58602)
metabase/metabase: docs/people-and-groups/saml-google.md

4. Paste the **User's first name attribute** under your Google **App attributes**.
5. Repeat steps 1-3 for the attributes "Last name" and "Email".

## Setting up synchronize group membership

> 1. In Google Admin, setup a multi-value [custom user attribute](https://support.google.com/a/answer/6208725?hl=en#zippy=%2Cadd-a-new-custom-attribute) for your users (recommended if you manage multiple SAML app permissions in Google or do not have existing Google Groups that align with your desired Metabase groups) OR [map to existing Google Groups](https://support.google.com/a/answer/11143403?hl=en).

Reviewer: ```suggestion
1. In Google Admin, set up a multi-value [custom user attribute](https://support.google.com/a/answer/6208725?hl=en#zippy=%2Cadd-a-new-custom-attribute) for your users (recommended if you manage multiple SAML app permissions in Google or don't have existing Google Groups that align with your Metabase groups) OR [map to existing Google Groups](https://support.google.com/a/answer/11143403?hl=en).
```
